### ISSUE TITLE
some bugs in torch_geometric/nn/models/mask_label.html

### ISSUE THREAD

**Issue by downeykking** [pyg-team/pytorch_geometric#5092]
### 🐛 Describe the bug

document and source code are here:
https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/nn/models/mask_label.html#MaskLabel
```python
def ratio_mask(mask: Tensor, ratio: float):
        r"""Modifies :obj:`mask` by setting :obj:`ratio` of :obj:`True`
        entries to :obj:`False`. Does not operate in-place.

        Args:
            mask (torch.Tensor): The mask to re-mask.
            ratio (float): The ratio of entries to remove.
        """
        n = int(mask.sum())
        out = mask.clone()
        out[mask] = torch.rand(n, device=mask.device) < ratio
        return out
```
Question: 
`ratio (float): The ratio of entries to remove.` Maybe this is the ratio of entries to keep?
And the example of mask_label is here:
https://github.com/pyg-team/pytorch_geometric/blob/3950ff617048948a5d89c8c78cd1d9408b47ad94/examples/unimp_arxiv.py
```python
def train(label_rate=0.65):  # How many labels to use for propagation.
    model.train()

    propagation_mask = MaskLabel.ratio_mask(train_mask, ratio=1 - label_rate)
    supervision_mask = train_mask ^ propagation_mask

    optimizer.zero_grad()
    out = model(data.x, data.y, data.edge_index, propagation_mask)
    loss = F.cross_entropy(out[supervision_mask], data.y[supervision_mask])
    loss.backward()
    optimizer.step()

    return float(loss)
```

This is the result of MaskLabel.ratio_mask(train_mask, ratio=1 - label_rate) (0.35)
Epoch: 489, Loss: 0.8817, Val: 0.7297, Test: 0.7260
Epoch: 490, Loss: 0.8879, Val: 0.7289, Test: 0.7226
Epoch: 491, Loss: 0.8860, Val: 0.7296, Test: 0.7262
Epoch: 492, Loss: 0.8916, Val: 0.7296, Test: 0.7261
Epoch: 493, Loss: 0.8899, Val: 0.7285, Test: 0.7215
Epoch: 494, Loss: 0.8809, Val: 0.7278, Test: 0.7219
Epoch: 495, Loss: 0.8831, Val: 0.7298, Test: 0.7272
Epoch: 496, Loss: 0.8777, Val: 0.7301, Test: 0.7289
Epoch: 497, Loss: 0.8807, Val: 0.7292, Test: 0.7243
Epoch: 498, Loss: 0.8847, Val: 0.7272, Test: 0.7197
Epoch: 499, Loss: 0.8836, Val: 0.7289, Test: 0.7243

This is the result of MaskLabel.ratio_mask(train_mask, ratio=label_rate) (0.65)
Epoch: 489, Loss: 0.8469, Val: 0.7316, Test: 0.7302
Epoch: 490, Loss: 0.8439, Val: 0.7303, Test: 0.7248
Epoch: 491, Loss: 0.8418, Val: 0.7320, Test: 0.7309
Epoch: 492, Loss: 0.8428, Val: 0.7326, Test: 0.7337
Epoch: 493, Loss: 0.8425, Val: 0.7336, Test: 0.7318
Epoch: 494, Loss: 0.8518, Val: 0.7320, Test: 0.7289
Epoch: 495, Loss: 0.8388, Val: 0.7322, Test: 0.7300
Epoch: 496, Loss: 0.8493, Val: 0.7323, Test: 0.7324
Epoch: 497, Loss: 0.8532, Val: 0.7330, Test: 0.7308
Epoch: 498, Loss: 0.8465, Val: 0.7308, Test: 0.7266
Epoch: 499, Loss: 0.8530, Val: 0.7303, Test: 0.7241

the second way is similar to the original paper https://arxiv.org/pdf/2009.03509.pdf
thus `MaskLabel.ratio_mask(train_mask, ratio=label_rate)` and the `ratio` mean keep probability.

Refactor:
```python
in https://github.com/pyg-team/pytorch_geometric/blob/3950ff617048948a5d89c8c78cd1d9408b47ad94/examples/unimp_arxiv.py
def train(label_rate=0.65):  # How many labels to use for propagation.
    model.train()

    propagation_mask = MaskLabel.ratio_mask(train_mask, ratio=label_rate)
    supervision_mask = train_mask ^ propagation_mask

    optimizer.zero_grad()
    out = model(data.x, data.y, data.edge_index, propagation_mask)
    loss = F.cross_entropy(out[supervision_mask], data.y[supervision_mask])
    loss.backward()
    optimizer.step()

    return float(loss)
```
in document and source code https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/nn/models/mask_label.html#MaskLabel
```
def ratio_mask(mask: Tensor, ratio: float):
        r"""Modifies :obj:`mask` by setting :obj:`1-ratio` of :obj:`True`
        entries to :obj:`False`. Does not operate in-place.

        Args:
            mask (torch.Tensor): The mask to re-mask.
            ratio (float): The ratio of entries to keep.
        """
        n = int(mask.sum())
        out = mask.clone()
        out[mask] = torch.rand(n, device=mask.device) < ratio
        return out
```

### Environment

PyG version: 2.0.4
PyTorch version: 1.1.10
OS: linux
Python version: 3.9
CUDA/cuDNN version: 10.2
How you installed PyTorch and PyG (conda, pip, source): pip
Any other relevant information (e.g., version of torch-scatter):
torch-cluster 1.6.0
torch-geometric 2.0.4
torch-scatter 2.0.9
torch-sparse 0.6.13
torch-spline-conv 1.2.1

**Comment by rusty1s** [comment]
I think you are absolutely right, let me try to fix that.
